routers/private/serv.go

```diff
@@ -265,6 +276,48 @@ func ServCommand(ctx *macaron.Context) {
 		}
 	}
 
+	// We already know we aren't using a deploy key
+	if !repoExist {
+		if user.IsOrganization() && !setting.Repository.EnablePushCreateOrg {
```

Comment:
Shouldn't this be `owner` rather than `user`? I may have gotten a little confused with this user/owner switching.

Reply:
You are correct, it should be owner. 😅  
Will fix in next push.